tycho-its/src/test/java/org/eclipse/tycho/test/issue1093/Issue1093.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Holger Voormann and others.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.tycho.test.issue1093;
+
+import java.util.List;
+
+import static org.junit.Assert.fail;
+
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.eclipse.tycho.test.AbstractTychoIntegrationTest;
+import org.junit.Test;
+
+public class Issue1093 extends AbstractTychoIntegrationTest {
```

Comment:
```suggestion
public class Issue1093Test extends AbstractTychoIntegrationTest {
```
The class should end with Test to be picked up by the surefire plugin.